patches/tModLoader/Terraria.ModLoader.Engine/Steam.cs

```diff
@@ -14,6 +14,8 @@ internal class Steam
 		public const uint TMLAppID = 1281930;
 		public const uint TerrariaAppID = 105600;
 
+		public const float SteamCloudQuotaLimitRatio = 0.9f;
```

Comment:
Why 90%? You should be able to fill until 100%

Reply:
since steam cloud doesn't compress, I could figure out the size of the file and add it up to available, and compare to total?